src/components/ChatList/elements/ChannelContent.style.ts

```diff
@@ -9,22 +9,24 @@ export const channelContentStyles = StyleSheet.create({
     fontSize: normalize(14),
     lineHeight: 22,
     flex: 1,
-    paddingBottom: 2
+    paddingBottom: 2,
+    color: COLORS.white2
   },
   description: {
     fontFamily: fonts.inter[400],
     fontSize: normalize(14),
     lineHeight: 22,
     alignSelf: 'center',
     flex: 1,
-    marginRight: 4
+    marginRight: 4,
+    color: COLORS.gray500
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the description text has been changed to `COLORS.gray500`. As with the previous comment, ensure that this color change is consistent with the overall theme and accessibility standards.

```diff
-    marginRight: 4
+    marginRight: 4,
+    color: COLORS.gray500
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->